src/Sulu/Component/Rest/FlattenExceptionNormalizer.php

```diff
@@ -22,28 +24,37 @@ class FlattenExceptionNormalizer implements NormalizerInterface
     /**
      * @var NormalizerInterface
      */
-    private $normalizer;
+    private $decoratedNormalizer;
 
     /**
-     * @var bool
+     * @var TranslatorInterface
      */
-    private $debug;
+    private $translator;
 
-    public function __construct(NormalizerInterface $normalizer, string $debug)
-    {
-        $this->normalizer = $normalizer;
-        $this->debug = $debug;
+    public function __construct(
+        NormalizerInterface $decoratedNormalizer,
+        TranslatorInterface $translator
+    ) {
+        $this->decoratedNormalizer = $decoratedNormalizer;
+        $this->translator = $translator;
     }
 
     public function normalize($exception, $format = null, array $context = [])
```

Comment:
method is called by the `SerializerErrorRenderer`: https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/ErrorHandler/ErrorRenderer/SerializerErrorRenderer.php#L74-L77